src/server/http/__tests__/base_path_rewrite.js

```diff
@@ -0,0 +1,188 @@
+import { Server } from 'hapi';
```

Comment:
__nit:__ not sure what is our convention here, but shouldn't this test file have the same name as the file it tests?